### DISCUSSION TITLE
Request for API documentation

### DISCUSSION THREAD

**Discussion by SXHRYU** [encode/django-rest-framework#9491] [Ideas & Suggestions] [answered]
I'd like documentation about [`format`](https://github.com/encode/django-rest-framework/blob/master/rest_framework/renderers.py#L48) attribute of renderers.

[Documentation page](https://www.django-rest-framework.org/api-guide/renderers/) doesn't explain what this attribute is for and what values for it are viable. I've found only one usage of this attribute in [source code](https://github.com/encode/django-rest-framework/blob/master/rest_framework/negotiation.py#L85-L86). It will still be great if this parameter's usage is explicitly documented.

**Comment by browniebroke** [comment, marked as answer]
The value comes from here:

https://github.com/encode/django-rest-framework/blob/master/rest_framework%2Fnegotiation.py#L40-L45

So I think it's when you call your endpoint like this:

```
/v1/users/1.json  # suffix is json
/v1/users/1.xml   # suffix is xml
/v1/users/1?format=xls  # format query param is xls
```

And so on... 

DRF can use this format override to filter suitable renderers, in the piece of code that you found.
